pages/account/manage/links.js

```diff
@@ -77,6 +77,12 @@ export default function ManageLinks({ BASE_URL, username, links }) {
         {success && (
           <Alert type="success" message="Link Created/Updated Successfully" />
         )}
+        {deleted && (
+          <Alert
+            type="success"
+            message={"Link Deleted Successfully"}
+          />
+        )}
```

Comment:
I think this could be combined into a single alert using a data object

```js
const alerts = {
   delete: {
      type: "success",
      message: "Link deleted successfully"
   }
   // ...
}
```